relay/relay-install.sh

```diff
@@ -72,6 +74,9 @@ RestartSec=7
 WantedBy=multi-user.target
 EOF
 
+# Create second relay systemd unit file with changed LOWEST_PORT and HIGHEST_PORT env
+sed /etc/systemd/system/firezone-relay-1.service 's/49152/57344/; s/57343/65535/' > /etc/systemd/system/firezone-relay-2.service
```

Comment:
I think replacing `LOWEST_PORT=49152` with `LOWEST_PORT=57344` would be easier to understand instead of just the port numbers.

Reply:
Changed